Makefile

```diff
@@ -81,7 +81,7 @@ mdlint: # Markdown lint for web
 .PHONY: web_redirects
 web_redirects:
 	# Generate netlify _redirects
-	find ./docs/ -path "*/*-*.md" | xargs grep -h slug | cut -d ' ' -f2 | awk -F '/' 'NF>2{print "/"$$2"/*",$$0,307}' > ./website/static/_redirects
+	find ./docs/ -path "*/*-*.md*" | xargs grep -h slug | cut -d ' ' -f2 | awk -F '/' 'NF>2{print "/"$$2"/*",$$0,307}' | sort -t '/' -k 2 > ./website/static/_redirects
```

Comment:
From Docusaurus docs:
> "For the most accurate parsing and better editor support, we recommend using the .mdx extension for files containing MDX syntax."

Reply:
Yes I know, it's just a recommandation. In our case, we have a majority of .md file. Since it's works with .md let's keep it coherent and use the same file extension.

Reply:
That's the thing, at least in my case it doesn't work. Having React components inside the .md file breaks my editor and it does not parse correctly (maybe it also breaks editors of other contributors).
I agree that consistency is important and tried to make it work with .md, but was not able to.
I think that the time spent in searching for a solution to keep it as .md is not worth it if it already works.

Reply:
Ok, let's have a look tomorrow together to identify why it breaks your editor. Maybe I'm missing something but it's works perfectly on my side.